snap/snapcraft.yaml

```diff
@@ -3,315 +3,105 @@ summary: Live streaming and video recording software
 description: Free and open source software for video capturing, recording, and live
   streaming.
 version: "31.0.3"
-base: core22
+base: core24
 grade: stable
 confinement: strict
 license: GPL-2.0
-architectures:
-  - build-on: amd64
 compression: lzo
 
+platforms:
+  amd64:
```

Comment:
Does upstream not support `arm64`?

Reply:
they currently don't build ARM binaries for linux